datastore/importers/nmdb-import-ipconfig/Parser.cpp

```diff
@@ -36,21 +36,31 @@ Parser::Parser() : Parser::base_type(start)
     // Skip garbage before
     *(!qi::lit("Windows IP Configuration") >> -qi::omit[+token] >> qi::eol) >>
     (qi::lit("Windows IP Configuration") > +qi::eol >>
-     +(-compartmentHeader >>
-       +( ("Host Name" >> dots > fqdn > qi::eol)
-            [(pnx::bind(&Parser::addDevInfo, this, qi::_1))]
-        | ("Primary Dns Suffix" >> dots > -fqdn > qi::eol)
-        | ignoredLine
-       ) >> qi::eol >>
-       *(adapter)
-      )
-    ) [(qi::_val = pnx::bind(&Parser::getData, this))] >>
+     (
+      +adapter |
+      +(
+        
+        -compartmentHeader >>
+        hostData >>
+        *(adapter)
+      
+       )
+     )
+    ) [qi::_val = pnx::bind(&Parser::getData, this)] >>
```

Comment:
This logic is wrong.  I sent you two sample files.  While it parses the first, it fails to parse the second config.  Use `-v9` to trace it; it appears to be an issue with the second adapter processing.